registration_service/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.17 as build-env
+
+WORKDIR /app
+
+COPY go.mod .
+#COPY go.sum .
+
+RUN go mod download
+
+COPY . .
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -installsuffix cgo -o /go/bin/registration_service
+
+FROM alpine:3.13
```

Comment:
Take a look at the golang hot reload that Sirius have in place. Useful here: https://github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/blob/main/docker/sirius-deputy-hub/Dockerfile#L26

Reply:
Lifesaver - rebuilding the container was getting realllllly annoying.

Reply:
Implemented ✅